.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please put all your node files in a .gitignore folder! There are hundreds of files here